layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
         "i" 'geiser-repl-interrupt
+        "f" 'geiser-load-file
+        "l" 'geiser-insert-lambda
         "m" 'geiser-repl-import-module
         "u" 'geiser-repl-unload-function
-        "h" 'geiser-doc-symbol-at-point))))
+        "hd" 'geiser-doc-symbol-at-point
+        "s" 'geiser-squarify
+        "q" 'geiser-repl-exit
+        "hc" 'geiser-customize-implementation)
```

Comment:
I don't recall we have added any binding for customizing a particular mode.

Is this a common part of workflow?

If you really want, something in ", T" is better because that's the prefix for toggles.

Reply:
You mean `, T` for toggle squarify? Or to go to customize?

Reply:
no, I mean geiser customize implementation.

Reply:
it's just not conventional. We haven't added similar bindings to other major modes.
If `SPC m h c` is a good binding for customization, it should also be applied to other major modes to maintain consistency.

It's better to open a separate PR for this. We need to be more cautious when devising a new convention.